### PR TITLE
Bump pipenv pin plette

### PR DESCRIPTION
**Pull request by oz123**

### CODE REVIEW

File: profiles/package.mask [hunk 1]
```diff
@@ -65,12 +65,15 @@ dev-java/ant-xz
 dev-java/gnu-jaf
 dev-java/failureaccess
 
+# Oz Tiram <oz.tiram@gmail.com> (2024-03-10)
+# plette in version 1.0.0 is broken and pipenv is still
+# not using it. Bug #925611
+=dev-python/plette-1.0.0
```
Comment (mgorny) [inline profiles/package.mask:71]: You're missing empty line here. Also, I don't get the point of masking it if you pin to it anyway.

File: profiles/package.mask [hunk 2]
```diff
@@ -65,12 +65,15 @@ dev-java/ant-xz
...
 # Michał Górny <mgorny@gentoo.org> (2024-03-10)
 # Packages that still require <dev-python/cython-3 and do not promise
 # a fix anytime soon.
 #
 # dev-python/python-efl: econnman as revdep, bug #925692.
-# dev-python/pipenv: via <pydantic-2, + dparse, bug #925698, bug #925553.
```
Comment (mgorny) [inline profiles/package.mask:73]: This seems to be the wrong commit for this.

**Review by mgorny — CHANGES_REQUESTED** [review]

File: profiles/package.mask (hunk 1 above)
Comment (oz123) [reply profiles/package.mask:71]: OK. You are correct we can do with pinning plette to 0.4.4 only.

File: profiles/package.mask (hunk 2 above)
Comment (oz123) [reply profiles/package.mask:73]: I moved unmask to a separate commit. I hope I understood you. Maybe it should be with the same commit as adding 2023.12.1? I can squash those commits if necessary.

**Review by mgorny — APPROVED** [review]
Thanks!

**Pull request closed without merging**